crates/embucket-lambda/Makefile

```diff
@@ -11,11 +11,12 @@ build:
 
 # Deploy to AWS (must run from workspace root for include paths to work)
 deploy: build deploy-only
-	@echo "Deployed $(FUNCTION_NAME) to AWS"
+    @echo "Deployed $(FUNCTION_NAME) to AWS"
 
 # Quick deploy without rebuild
 deploy-only:
-	cd ../.. && cargo lambda deploy --binary-name bootstrap $(FUNCTION_NAME) --env-file $(ENV_FILE)
+	cd ../.. && cargo lambda deploy --layer-arn arn:aws:lambda:us-east-2:184161586896:layer:opentelemetry-collector-arm64-0_19_0:1 arn:aws:lambda:us-east-2:767397688925:layer:otel-collector-config:19
```

Comment:
otel-collector-config is extra layer that from one side hides complexity hiding config file. But that add extra dependency on layer, that should be maked public to be used by our users. This layer potentially may have wtf effect. As much simpler and transparent would be dding this config just to lambda deploy, so user will not be even thinking about somebody send its tracing somewhere else.
It's possible to make otel-collector-config part of our deploy, and we should consider this option before merging this.